tests/integration/it_reservations_test.py

```diff
@@ -1,42 +1,45 @@
-from datetime import datetime
+from http import HTTPStatus
 
 from flask import url_for
 
 import pytest
-from timeless.restaurants.models import Reservation, ReservationStatus
-
-def test_create_reservation(client, db_session):
-    reservation = Reservation.query.get(100)
-    if reservation:
-        db_session.delete(reservation)
-        db_session.commit()
-
-    comment = "Comment for verification"
-    response = client.post(url_for("reservations.create"), data={
-        "id": 100,
-        "start_time": datetime(2019, 2, 20, 21, 30),
-        "end_time": datetime(2019, 2, 20, 22, 30),
-        "customer_id": 1,
-        "num_of_persons": 2,
-        "comment": comment,
-        "status": ReservationStatus.confirmed.name
-    })
-    assert response.location.endswith(url_for("reservations.list_reservations"))
+from tests import factories
+from timeless.restaurants.models import Reservation
+
+
+@pytest.mark.skip
```

Comment:
@vladarefiev can you add the reason for skipping this test? I think you can do it by adding paranthesys: ``("reason for skipping")``

Reply:
@amihaiemil okay